src/web/client/extension.ts

```diff
@@ -166,6 +164,8 @@ export function activate(context: vscode.ExtensionContext): void {
                                     }
                                 );
 
+                                powerPagesNavigation();
```

Comment:
what is the change in experience with this? the Actions hub shows "no data provider regsitered" till all files are fetched?
 
Alternately, can we await this function instead and then proceed to fetching files? What is the experience in that case? Is the delay too long?

Reply:
![image](https://github.com/user-attachments/assets/7ffffcd4-562f-44c0-9455-9a4efe5cb995)
Until the user is authenticated and before all files load, this message will be shown if the user has opened the Power Pages actions.
Resolving this comment, as this is not hard requirement, will come up later for better experience

Reply:
this option looks good - we should consider long term experience with design on this